assets/scss/abstracts/_variables.scss

```diff
@@ -1,18 +1,17 @@
 :root {
-  --middle-blue-green_40: hsla(174, 64%, 71%, 0.4);
-  --midnight-green: hsl(186, 100%, 19%);
-  --midnight-green_a25: hsla(186, 100%, 19%, 0.25);
-  --independece: hsl(236, 14%, 39%);
-  --mint: hsl(182, 100%, 35%);
-  --space-cadet: hsla(226, 45%, 24%);
-  --eerie-black: hsl(0, 0%, 13%);
-  --alice-blue: hsl(187, 25%, 94%);
-  --gainsboro: hsl(0, 0%, 87%);
+  --color-primary-200: hsl(182, 100%, 35%);
```

Comment:
The numbers at the end of colors do mean something, based on how eg. material design does that. You typically start with the middle number 500 for the middle color, and for the brighter ones you go up -100, for the darker ones you go down +100.

So eg. `--color-secondary-300: hsla(0, 0%, 100%, 0.2);` should be one of the neutrals